src/frame/vulkan.rs

```diff
@@ -32,91 +32,109 @@ pub struct Vulkan {
 impl Vulkan {
     pub fn new() -> Result<Self, Box<dyn Error>> {
         let app_name = CString::new("wluma")?;
-        let app_info = vk::ApplicationInfo::builder()
-            .application_name(&app_name)
-            .application_version(WLUMA_VERSION)
-            .engine_name(&app_name)
-            .engine_version(WLUMA_VERSION)
-            .api_version(VULKAN_VERSION);
+        let app_info = vk::ApplicationInfo {
+            p_application_name: app_name.as_ptr(),
+            application_version: WLUMA_VERSION,
+            p_engine_name: app_name.as_ptr(),
+            engine_version: WLUMA_VERSION,
+            api_version: VULKAN_VERSION,
+            ..Default::default()
+        };
 
         let instance_extensions = &[
-            vk::KhrExternalMemoryCapabilitiesFn::name().as_ptr(),
-            vk::KhrGetPhysicalDeviceProperties2Fn::name().as_ptr(),
+            CStr::from_bytes_with_nul(b"vkGetPhysicalDeviceProperties2\0")
+                .unwrap()
+                .as_ptr(),
         ];
```

Comment:
I think you might have lost one of the two extensions here, and from the patch you linked earlier today I think they found a more correct patch in this case:

![image](https://github.com/user-attachments/assets/cba8e41a-69e6-4acd-8161-2d91df22a2aa)